### ISSUE TITLE
TypeError: Cannot read property 'failureType' of null

### ISSUE THREAD

**Issue by mhalttu** [dubocr/homebridge-tahoma#354]
I'm seeing this error in the logs right before a SIGTERM, causing the bridge to shutdown.

```
15/01/2022, 19:02:02 TypeError: Cannot read property 'failureType' of null
    at Execution.onStateUpdate (/home/hoobs/.hoobs/node_modules/overkiz-client/src/models/Execution.ts:54:19)
    at Timeout._onTimeout (/home/hoobs/.hoobs/node_modules/overkiz-client/src/Client.ts:212:31)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
```

**Comment by dubocr** [comment]
Hi,

It should be fixed in 2.2.15.

**Comment by mhalttu** [comment]
Thank you!
